classes/01class/exercises/c01-git01/mail2kvanitha/README.md

```diff
@@ -0,0 +1,33 @@
+# c01-git01
```

Comment:
This file should not be part of the c01-git03 exercise delivery

Reply:
Removed c01-git01 file from c01-giit03 branch

Reply:
projects/project02/application/web/public/favicon.ico file removed, Please advise if there is anything pending.